workshop/exercises/pygeoapi.config.yml

```diff
@@ -184,6 +184,37 @@ resources:
               data: tests/data/ne_110m_admin_0_countries.sqlite
               id_field: ogc_fid
               table: ne_110m_admin_0_countries
+    airports:
+        type: collection
+        title: airports of the world
+        description: airports of the world
+        keywords:
+            - airports
+            - natural earth
+        links:
+            - type: text/html
+              rel: canonical
+              title: information
+              href: https://www.naturalearthdata.com/
+              hreflang: en-US
+        extents:
+            spatial:
+                bbox: [-20037508.34, -20048966.1,20037508.34, 20048966.1]
```

Comment:
Is this bounding box correct? You can check the correct bounding box in QGIS

Reply:
Is the dataset covering the entire world? If not, you should use the bounds of the dataset.